reflex/components/component.py

```diff
@@ -2168,6 +2169,7 @@ def _render_stateful_code(
             tag_name=tag_name,
             memo_trigger_hooks=memo_trigger_hooks,
             component=component,
+            positions=constants.Hooks.HookPosition,
```

Comment:
this is kind of hacky. constants should be added to the global `consts` namespace in `reflex/compiler/template.py`